05_stringer/mohankrishna/main_test.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var tests = []struct {
+	input  IPAddr
+	output string
+}{
+	{IPAddr{1, 2, 2, 2}, "1.2.2.2"},
+	{IPAddr{1}, "1.0.0.0"},
+	{IPAddr{}, "0.0.0.0"}}
```

Comment:
Put closing `}` for the array on a new line

Reply:
When I move the last `}` to a new line, linter throws an error saying `missing ',' before newline in composite literal`. Putting a `,` at the end and moving `}` to a new line works fine. But I thought this is more readable than adding a comma just for a new line.

Reply:
This is a matter of taste. @anzdaddy always requests:
```go
{
	{IPAddr{1, 2, 2, 2}, "1.2.2.2"},
	{IPAddr{1}, "1.0.0.0"},
	{IPAddr{}, "0.0.0.0"},
}
```
I think because
- you can add an extra line without modifying the last
- it's consistent with the rest of the curly usage in go.